.github/workflows/python.yml

```diff
@@ -120,6 +120,7 @@ jobs:
     env:
       ARROW_HOME: /usr/local
       ARROW_DATASET: ON
+      ARROW_SUBSTRAIT: ON
```

Comment:
Can we keep this alphabetically-ordered?

Reply:
Ah yes. I will fix